### PR TITLE
Style/basic format style

### PR DESCRIPTION
**Pull request by GraceGardner**
____ Wrote Tests ___x_ Implemented ____ Reviewed

## Neccesary checkmarks:

    [] All Tests are Passing in all environments

    [x] The code will run locally

## Type of change

    [x] New feature
    [] Bug Fix
    [] Refactor

## Description of Change:

    description: Added styling and basic elements/classNames 
    
## Requested feedback: 
    
    I'd like feedback on... 

## Check the correct boxes

    [x] This broke nothing
    [] This broke some stuff
    [] This broke everything

## Testing Changes

    [x] No Tests have been changed
    [] Some Tests have been changed
    [] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:

    [x] My code has no unused/commented out code
    [] My code has no binding.pry's
    [] I have reviewed my code
    [] I have commented my code, particularly in hard-to-understand areas
    [] I have fully tested my code

(For Fun!)Please Include a link to a gif of your feelings about this branch

Link:

### CODE REVIEW

File: src/Components/AllSightings.js [hunk 1]
```diff
@@ -1,11 +1,69 @@
...
+
+  const toggleDisplay = () => {
+    setDisplay(!display)
+  }
+
+
   return (
-    <p>All Sightings</p>
+    <>
+      <div className='search-container'>
```
Comment (krdurrett) [inline src/Components/AllSightings.js:16]: Nice work adding these sightings card!

File: src/Components/Card.js [hunk 2]
```diff
@@ -1,10 +1,15 @@
-import './Card.scss';
+import '../Styles/Card.scss';
 import React from 'react';
+import bigfoot from '../assets/bigfoot.jpg'
 
 const Card = () => {
   return (
-    <p>Card</p>
+    <div className='card'>
```
Comment (krdurrett) [inline src/Components/Card.js:7]: Love this initial framework for a card

File: src/Components/Home.js [hunk 3]
```diff
@@ -36,9 +39,40 @@ const Home = () => {
 
   return (
     <>
-      <p>Home Page</p>
+    <div className='home-message-container'>
+      <p className='home-message'>
```
Comment (krdurrett) [inline src/Components/Home.js:43]: The home page is laid out exactly like the wireframe

File: src/Components/NavBar.js [hunk 4]
```diff
@@ -9,15 +9,20 @@ const NavBar = () => {
   const { setClick, clicked } = useContext(NavigationContext);
   return (
     <>
-      <Link to={'/'}>
-        <p>OK, CRYPTID</p>
-        <img src={logo} alt='amazing sasquatch in the woods logo' className='logo' />
+    <div className='nav-bar'>
```
Comment (krdurrett) [inline src/Components/NavBar.js:12]: Love the nav bar

**Review by krdurrett — APPROVED** [review]
The styling looks lovely!

**Review by RosaTheDev — APPROVED** [review]

**Review by jasmyn2244 — APPROVED** [review]
Thanks Grace for doing this styling. It looks great! I especially love the golden annimation

**Pull request merged**